### PR TITLE
added german language with aditional emojis

### PR DESCRIPTION
**Pull request by ALMerrill**
Added german language file along with extra emojis that another PR added to english and a couple other languages

### CODE REVIEW

File: lang/de.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    {
+      name: 'geist',
+      emoji: '👻',
+    },
+    {
+      name: 'halloween',
+      emoji: '🎃',
+    },
+    {
+      name: 'abend vor allerheiligen',
```
Comment (LucasCtrl) [inline lang/de.js]: Sorry, at the moment multi-part words don't work (see issue #9)

**Review by LucasCtrl — CHANGES_REQUESTED** [review]

**Comment by LucasCtrl** [comment]
Maybe I can find a solution for the duplicate lines 34 and 38 by following issue #10.

File: lang/de.js (hunk 1 above)
Comment (ALMerrill) [reply lang/de.js]: Fixed

**Review by LucasCtrl — APPROVED** [review]

**Comment by LucasCtrl** [comment]
The bot will be updated during the night with the new translation file 😉

**Comment by ALMerrill** [comment]
Nice! I added it to my server to play around with it

**Comment by ALMerrill** [comment]
Also, where would I look in the code to add multi word phrases functionality? I could take a look at that

**Comment by LucasCtrl** [comment]
Referring to the issue #9 you could see inside this file https://github.com/LucasCtrl/spookyBot/blob/main/utils/reactMessage.js

**Pull request merged**